### PR TITLE
Attempt at fixing Github actions on stable

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Comment by Geod24** [comment]
I think we should just disable the multilib test, since it's so broken: https://issues.dlang.org/show_bug.cgi?id=19116

File: .github/workflows/runnable_cxx.yml [hunk 1]
```diff
@@ -138,7 +135,7 @@ jobs:
       with:
           # The host compiler does not matter too much, it is simply used for building DMD
           # Other CIs will test bootstrapping.
-          compiler: dmd-2.091.0
+          compiler: dmd-2.095.1
```
Comment (MoonlightSentinel) [inline .github/workflows/runnable_cxx.yml:138]: Maybe use ldc as the host compiler (as mentioned in the issue)?

```suggestion
          compiler: ldc-1.25.1
```

Comment (Geod24) [reply .github/workflows/runnable_cxx.yml:138]: Then the generated DMD won't segv, but the test suite will.

**Comment by Geod24** [comment]
@WalterBright : This will fix the issues you're seeing. However, 32 bits is broken because PIE is the default everywhere nowadays, see https://issues.dlang.org/show_bug.cgi?id=19116

**Pull request merged**